companies/views.py

```
@@ -487,6 +608,296 @@ def sync_company_from_dart(request, stock_code):
+                    results["skipped_count"] += 1
+                    continue
+
+                company_results = {}
+                company_errors = []
+
+                if sync_info:
+                    try:
+                        service = CompanyInfoService()
+                        service.sync_company_info(company)
+                        company_results["info"] = "동기화 완료"
+                    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)